vernac/vernac.mllib

```diff
@@ -1,6 +1,7 @@
 Vernacexpr
 Attributes
 Pvernac
+Prettyp
```

Comment:
You could also put this after `Declaremods` and thus remove all the hooks in `Prettyp`

Reply:
>You could also put this after Declaremods and thus remove all the hooks in Prettyp

What hooks?

Reply:
The `mod_ops` and `library_accessor` functions.

Reply:
OK